template/App.tsx

```diff
@@ -26,28 +27,35 @@ import {
   ReloadInstructions,
 } from 'react-native/Libraries/NewAppScreen';
 
+import 'react-native/tvos-types.d';
```

Comment:
Is this required? Afaik typings should be included in the `.tsconfig`.

Reply:
That is needed to pull in some types that only exist in the TV repo (focus management etc.)